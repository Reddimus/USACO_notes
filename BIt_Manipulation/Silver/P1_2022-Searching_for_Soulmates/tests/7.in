10
1703358258 3040091064
4149381102 1754389053
3508638470 1388587262
203102697 1327958277
2694198129 644904352
2395517296 326827326
592157688 1922686134
3070260618 3872028968
686755775 3123184895
465344489 3473849828
