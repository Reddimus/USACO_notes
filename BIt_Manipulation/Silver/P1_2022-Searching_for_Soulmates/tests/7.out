87
86
91
79
87
80
81
88
74
86
